cogs/greetings.py

```diff
@@ -3,22 +3,26 @@
 import os
 from discord.ext import commands
 
+# Class dedicated to greeting new members when joining the server
 class Greetings(commands.Cog):
+
+# Constructor that assigns the welcome message to the specific new member with their username
     def __init__(self, client):
         self.client = client
         self.welcome_message_id = self.load_welcome_message_id()
-
+# Helper function responsible for loading and returning the welcome message
     def load_welcome_message_id(self):
         try:
             with open("txtfiles/welcome_message_id.txt", "r") as file:
                 return int(file.read())
         except FileNotFoundError:
             return None
-
+# Helper function responsible for retaining the welcome message specific to the user
```

Comment:
Besides just what a why might be a little helpful here. 
IE:
# Helper function storing welcome message id to be loaded in the future if bot goes offline

Reply:
I have added that the bot will retain the id for the future and load it when online again.